NAMESPACE

```diff
@@ -1,16 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
-export(add_obs_error_to_nowcast)
-export(add_obs_errors_to_nowcast)
+export(add_obs_err_to_pt_nowcast_mat)
+export(add_obs_errs_to_pt_nowcast_mat)
 export(aggregate_df_by_ref_time)
 export(apply_delay)
-export(convert_reporting_square_to_df)
+export(convert_nowcast_matrix_to_df)
 export(estimate_dispersion)
-export(generate_point_nowcast)
-export(generate_point_nowcasts)
+export(generate_pt_nowcast_mat)
```

Comment:
do you prefer having full generate vs shortening to gen and having point vs pt?

Reply:
I have used pt throughout, think generate is fine but can also shorten